firmware/lm32/heartbeat.h

```diff
@@ -0,0 +1,21 @@
+#ifndef __HEARTBEAT_H
+#define __HEARTBEAT_H
+
+#define HDMI_IN0_FRAMEBUFFERS_BASE 	0x01000000
```

Comment:
Don't redefine these here, move them out of the hdmi_inX.c files into the hdmi_inX.h files and then include those.